dist/recursive-fonts.js

```diff
@@ -13,149 +13,214 @@
9ItL5kab61cq9sRnxSwwQ1sa3/xB/ojfr408X5nS+mTZ7MZbG1jVJq1pxLrxx6k+RT3G+us2AoAawsVnHHzz6lAohldZjhQKbrDs/1Bj60gIcj0iNKkdTqYyoigAwytuxNwxPT7wqRDNyRFBVAItMaCIv9oQxQgWblwIXrDX2br5PvAlUYYVDaRnGNJqXBQUiGkeMIMSAiOnpyVPrHWlrjqwwqlr4YnFBMPIG9cXjdsYoZSyu7IOqaRxUy+sFG7pTuVnNzfePz9SCdh/EUDSZGzBhL+PTeOXP1A5Ba0EvAqoX8fk2XKCvywXoXy/x4LtGMoN6n3pKGCAud863Fyv5z2XvK0y0UWkCmuB4bRfLKkcSh9hzKKFOaK6Jb0WabAh9urEFJ353XBMXwEgV/0iTMAO0x8N6BiUexPdH/uTi8fAM2nJX6wiI54204iUnNLbCL9jzfvTGXXCtPIK0OIa/ek8fmHERFPbW+1MzFgqCGHxjx+/avSZSS8tQTTTPxTvjEoil7zjPo96+CSXLGYJBPLZdfnNILG0k1E5ZbSqRjPFNsCJJkc4QJ2fp87i0jqXl+e3tF7Vgs96WBCMIj/JzXXjlYtXLfiGUzO4H7fyMahn5V+oqwMQsxsOgsrXDr6D/+HSOnTmM/Qpiq+VWIVZV24jc2u0Xjbcb37UfOWIr4xQGBsD6ViA/Ru4W/raW5m/AvLeozhi8ZzT+q2ey8cUAYFRHCg+oc0WEv3EVFxt+kP3QywVvZFu++IjeWxbUWFJD2kUMx0wti9JbvBJZBmuVVsk9jNeWoXN4wcrcwU1wvsxlViONNEKElgoULGPpJJeRt4+Na6DvboOsO/6k5qyIAvyqboY79AhvYjVL1tRF9rtGjG6DmHbTmpMaaXn28AFYx8WXb7o1ZM2fgA4MqNPJUCHNgTgsOZ9u1ftCdhf9214J4R1K4SalRLqG0NAaybYVP/J1+/HyPOYhd4Sa69H4iw34eKOnghbh9eWLdS10Q3CKe/QdqlHe/DPMbjVTdpaS9byuNb9KMLn66oZ1RWxn6WdPd+1OVPdvQ/BQ3DuIY+50L6sE1KqWzTp7sV9g+H7j8gtzaA1T8XMvcJv7stocPfNZhTuKO/RJpCyOWqVKBH7tXUBYYJ2Ch1VEjaJx8WLbSwrFNytRjHe6VkKtGDx63hm6rB7aPFkBcUE/9cLNoqSnNHuRTyKWhT1Yfmk59JJRiQMiArDia/u5j9wasXxsC8vOpuRpkXgkMoncEKf5qJCMCjdg0ZQzkGZ306lwhPO+FlveLs0Kk9WLj94rLbMwv7QfnT/Bgairl+6OsLG2c3MO699craOpjkMcJWOREk7vDJXmbhhN/RkEr0okj71kgoZUnU7oOZZJJvQ906TpltY4ULau/+oN6evPXyyFvYu5JmEtTyhoGAaKr8IoCAbVgGXPNu4gXIF82lEE5nVY52Nl8OZZ4nlZVQ/8knneMNKJInaKftSRrcub3IdmxzR2bRar7eWldh9rjtW4ZEscWqtmkUMZVbpwFtCDFruXzY1geoBlYaUtn4hXo2ybzcVF/KR434nrKcK89zUCtKTldtrXF8ujDsOXm6AAnenhph9Y98OX8/agtJSCoVzgLVZuKtsEFbD+TWz3XnqDemh1hk1ZiOjdcg/MDKSzEUIksP4F6GxUud7F9VM7iuGBxmp6Vm9WXdvbbNbQU3MjEc4nn7lQQGg9QBODOB8hJs+96ED4M+n/H3pDppzwJ4H6rsuEGD0c93geUEWMYYQQWVOkfkLALBfUpBBnjTatKyac3eqO8zw4dcqLdFw+63im9+9Jo+//4pe7LY+KwwvewAlvi02d1qegiqbYod3zCU7qRkybr/fQCuZAYePOVjgOz2nn+F4AEGURnFbkM6gjUTU4uAIlil1QfCB5/K5McDhhCHyx6QffwbpthYLALez2tWFX9IpsJvJ1EXPzPvEeSCw1YjB1OftHTWbxnPQrEdwtTgS6PG3aO9XQ0VvcKdjtp7vSFFY8fv+ctj2f1ZYt/u9/R1WTA8CzaAccCTkQzCbAESzOkMqNatvURmacxufT28i2YW1JzjTak5V0KBSOxUmaS8I0WrvTCZHcbgaxzG5ygwMm+OOhoJ07q8+VdnMuC6lEARTkyHd6MklGRMRKvFlWWrQp9RXjpP0RaX8chThOHQtQuvsoF8keAIKv0c4CqlO8UmM/5u8qEH4JmNSLkXW8LflEdD64eRaa/dRGXtWrkgO3L+8mcYIiPA8W/WJ5hQN5SflXH4ulXj6vfKXvGyDXAnJwfVINTk/ef3bO6frVgbZ9a7nj1u2fxdg1Uk7PjG7cN6NwOdpY0nhyciN7oP3Wh+lMzQqHiWTgdPap5uPNgsl6vvYX85v7swJ9MSlp4sDpcC2jvGMRnxGMQNUeiB0J4ywkJMzjxcJuWXa9HWfyfN/wEe68XbgOEXP8FmfOAvA8wQ1jazTp9xTGKLtXS+MFsZqpMoudL0CaHByen4oiz3PHn9EwmFYzrBVjr7eWEFNpZoY/hMvd55sIZu31qUSytUdaofWladYqxw85liEKaUOwj6QZlW1dl1k1iCBROea3cXdakyStav06ZrGQYoJX67FqT5ZgaTwg+PRp0zaFrX9jECFwPvQntiCGw6djGqKheLC5FMJ3bzhco5VOZ235c3XLTHqmh46F1mHfS9zEceXG8+sWz5ph6C2ntO2/LzEGszgK0NVlWBFuvtF2++VgXPqgmYBSCfodDAPsDxMbNfRf02EUEtGNMpEBztix3Lmkk7I/HMzMCDDmRWXwqBH2L/755d1ScCTrmVjcPOCimKUMe+UpQwA3LQOCiXSDdYtX+D0odHBer502R1UObSidRdO/W5QstIbv7YmJoCjUmeH2hTnY7ZNAjAdolxiCp0sQh3+0AtxkT1FtiMboS33StbAT+7PW3a0SPXptcacw2sO+ylM6+k9UHmkkUoxEVEe/r7F2M2fyjLYisMLqfaikCtsol7r3RxgK+uk
```

Comment:
@RoelN I'd like to update this to the latest font (and then I'll probably want to update it at least once more in the future). It has quite a few improvements and some small adjustments to the instance values along the weight axis. Did you generate the instance value dict, or just write it out? It may be fastest for me to write out the changes, but if there's a quick and accurate way, it would be cool to use that, too.

Reply:
No automated process — just a manual update of the axes min/max/default indeed. Feel free to pass the font.

I'll take the latest from https://github.com/arrowtype/recursive/releases and update this branch before merging!

Reply:
Done, moved from xprn → casl

Reply:
Thank you!